server/chat-plugins/help.ts

```diff
@@ -463,6 +473,7 @@ export const commands: ChatCommands = {
 		unban: 'ban',
 		ban(target, room, user, connection, cmd) {
 			this.room = Answerer.getRoom();
+			if (!this.room) return;
```

Comment:
For a command, it would be wise to give the user an error message rather than silently failing.